crates/ubrn_cli/src/ios.rs

```diff
@@ -190,10 +208,44 @@ impl IOsArgs {
         Ok(())
     }
 
+    fn lipo_when_necessary(
+        &self,
+        crate_: &CrateConfig,
+        target_files: HashMap<Target, Utf8PathBuf>,
+    ) -> Result<Vec<Utf8PathBuf>> {
+        let mut by_platform = HashMap::new();
+        for (target, file) in target_files {
+            let files = by_platform.entry(target.platform).or_insert(Vec::new());
+            files.push(file);
+        }
+        let metadata = crate_.metadata()?;
+
+        let mut sorted = Vec::new();
+        for (p, mut files) in by_platform {
+            if files.len() == 1 {
+                sorted.append(&mut files);
+            } else {
```

Comment:
Is lipo needed on android?  We haven't had to use it so far, but I guess they don't have separate 'simulator' targets?

Reply:
> Is lipo needed on android?

No, `lipo` is part of the Apple toolchain.